File: app/src/main/java/com/postulacion/githubjavapop/model/Repository.java
package com.postulacion.githubjavapop.model;

import com.google.gson.annotations.SerializedName;

public class Repository {
    @SerializedName("name")
    private String name;
    @SerializedName("description")
    private String description;
    @SerializedName("forks_count")
    private String forks;
    @SerializedName("stargazers_count")
    private String stargazers_count;
    @SerializedName("owner")
    private User user;

    public Repository(String name, String description, String forks, String stargazers_count, User user) {
        this.name = name;
        this.description = description;
        this.forks = forks;
        this.stargazers_count = stargazers_count;
        this.user = user;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getForks() {
        return forks;
    }

    public void setForks(String forks) {
        this.forks = forks;
    }

    public String getStargazers_count() {
        return stargazers_count;
    }

    public void setStargazers_count(String stargazers_count) {
        this.stargazers_count = stargazers_count;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }
}

File: GitHubJava/app/src/main/java/com/example/eloyvitorio/githubjavapop/data/model/Repository.kt
package com.example.eloyvitorio.githubjavapop.data.model

import com.google.gson.annotations.SerializedName

class Repository(
        @SerializedName("id")
        val id: Int,
        @SerializedName("name")
        val name: String,
        @SerializedName("full_name")
        val fullName: String,
        @SerializedName("owner")
        val owner: Owner,
        @SerializedName("description")
        val description: String,
        @SerializedName("stargazers_count")
        val stargazersCount: Int,
        @SerializedName("forks")
        val forks: Int
)